FusionMCPBridge/FusionMCPBridge.py

```
@@ -0,0 +1,674 @@
+        distance = float(distance)
+    through = bool(data.get('through', False))
+    task_queue.queue_task('holes', points, width, distance, faceindex, through)
+    return {"status": 200, "data": {"message": "Loch wird erstellt"}}
+
+def handle_threaded(path, method, data):
+    inside = bool(data.get('inside', True))
+    allsizes = int(data.get('allsizes', 30))
+    task_queue.queue_task('threaded', inside, allsizes)
+    return {"status": 200, "data": {"message": "Threaded Feature wird erstellt"}}
+
+def handle_circular_pattern(path, method, data):
```

Ruff (0.14.10)

Unused function argument: `path`

(ARG001)